python/lsst/afw/detection/peak.cc

```diff
@@ -111,11 +111,9 @@ PYBIND11_PLUGIN(_peak) {
     table::pybind11::declareCatalog(clsPeakCatalog);
 
     clsPeakRecord.attr("Table") = clsPeakTable;
-    clsPeakRecord.attr("ColumnView") = clsPeakColumnView;
```

Comment:
I see in the other commit that it does exist for `BaseRecord` why the discrepancy?